DATA/common/setenv.sh

```diff
@@ -160,6 +162,17 @@ has_matching_qc()
   has_detector_matching $1 && [[ $WORKFLOW_DETECTORS_QC =~ (^|,)"$1"(,|$) ]]
 }
 
+has_pid_qc()
+{
+    PIDDETECTORS=$(echo $1 | tr "-" "\n")
+    for PIDDETECTOR in $PIDDETECTORS;
+    do
+	echo PIDDETECTOR=$PIDDETECTOR 1>&2
+	! has_detector_qc $PIDDETECTOR ]] && return 1
```

Comment:
I assume that you have tested this and it works. But don't you need a matching `[[` for the `]]`?

Reply:
I did test it, I guess I made a typo later on (I was doing several tests yesterday in parallel). I will fix it. 
Thanks for spotting!

Reply:
Fixed! Thanks again!